packages/components/src/DataTable/Footer.css

```diff
@@ -0,0 +1,37 @@
+.footerContainer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  min-height: 52px;
+  margin: 0 16px;
+}
+
+.footerContainer .pageNavigation {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.footerContainer .pageNavigation span {
+  padding-left: 8px;
+  white-space: nowrap;
+}
+
+.footerContainer .pageNavigation .pageNavigationSelect {
+  min-width: 70px;
+}
+
+.footerContainer .pageNavigation .pageNavigationButtonsContainer {
+  display: flex;
+  width: 100%;
+
+  padding-left: 16px;
+  justify-content: space-between;
+}
+
+.footerContainer
```

Comment:
This is a super-brittle selector, can we add a classname or something else more targeted?

Reply:
Yeah, overall I believe we should revisit the CSS implementation, maybe we could have a 15-30min call just for us to know how to approach this if you think that makes sense.